backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -12,7 +14,8 @@ import java.time.ZoneOffset
 @Suppress("unused")
 class CardMutationService {
     @GraphQLDescription("Stores a new digital EAK")
-    fun addCard(card: CardGenerationModel): Boolean {
+    fun addCard(context: GraphQLContext, card: CardGenerationModel): Boolean {
+        if (!context.isSignedIn) throw GraphQLException("Unauthorized")
```

Comment:
maybe it would be nice, if we would throw sth like an actual `UnauthorizedException` and catch it in the handler and return a `401` status code... Or do we already do that magically?

Reply:
Done